modules/shadertools/src/modules/lighting/lights/lighting-uniforms.ts

```diff
@@ -34,7 +34,7 @@ export type PointLight = {
   position: Readonly<NumberArray3>;
   color?: Readonly<NumberArray3>;
   intensity?: number;
-  attenuation?: number;
+  attenuation?: Readonly<NumberArray3>;
```

Comment:
Is this is a breaking change? The getUniforms could normalize numbers to arrays.

Reply:
deck was not importing these types before and was always passing an array, I've added the luma types to do a cross check: https://github.com/visgl/deck.gl/pull/9074/files#diff-8380e342b2e1a1cb8e1a5b2f5ef19ccc69640d9d87f1f5c523112fec65ddd87dR12

This specifically isn't a breaking change: the shader ignored the attenuation completely before, now that it is implemented passing an attenuation may give different results from before